erpnext/accounts/report/accounts_receivable/accounts_receivable.html

```diff
@@ -21,11 +21,13 @@ <h5 class="text-center">
 				<th style="width: 40%">{%= __("Party") %}</th>
 				<th style="width: 15%">{%= __("Invoiced Amount") %}</th>
 				<th style="width: 15%">{%= __("Paid Amount") %}</th>
+				<th style="width: 15%">{%= report.report_name === "Accounts Receivable" ? 'Credit Note' : 'Debit Note' %}</th>
```

Comment:
the label should be translatable 

`report.report_name === "Accounts Receivable" ? __('Credit Note') : __('Debit Note')`